frontend/playwright.config.ts

```diff
@@ -108,10 +108,14 @@ const config: PlaywrightTestConfig = {
    outputDir: 'tests/playwright/test-results/artifacts/',
 
    /* Run your local dev server before starting the tests */
-   // webServer: {
-   //   command: 'npm run start',
-   //   port: 3000,
-   // },
+   webServer: {
+      cwd: '../',
+      command: 'pnpm run dev',
+      port: 3000,
+      timeout: 120 * 1000,
+      /* Reuse the same server if on local dev */
+      reuseExistingServer: !process.env.CI,
```

Comment:
Is this `webServer` going to start the `frontend` app (assuming the port is 3000)? If yes, then we need to have both frontend and backend to be running. If not, then I believe the port for the backend is different.

Also if a dev server is already running for the frontend app, does this mean Playwright won't start a new one and reuse an existing server?